web/modules/custom/dbcdk_community/src/Form/ProfileEditForm.php

```diff
@@ -153,10 +149,10 @@ public function buildForm(array $form, FormStateInterface $form_state, $username
     ];
 
     $form['birthday'] = [
-      '#type' => 'date',
+      '#type' => 'datetime',
```

Comment:
Why is using a `datetime` element better than using a `date` for representing birthdays in a form?